sdk/eventgrid/eventgrid/review/eventgrid.api.md

```diff
@@ -1071,12 +1084,23 @@ export interface PolicyInsightsPolicyStateDeletedEventData {
     timestamp: string;
 }
 
+// @public
+export type RecordingChannelType = string;
+
+// @public
+export type RecordingContentType = string;
+
+// @public
+export type RecordingFormatType = string;
+
 // @public
 export interface ResourceActionCancelEventData {
-    authorization: string;
-    claims: string;
+    authorization: ResourceAuthorization;
+    claims: {
+        [propertyName: string]: string;
+    };
```

Comment:
nit, small preference for `Record<string, string>` instead of the index signature, same in other places